### PR TITLE
Adds some instructions on how to make a model.py file

### PR DESCRIPTION
**Pull request by zjgarvey**

### CODE REVIEW

File: e2eshark/onnx/models/README.md [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+```
+
+### 1. Figure out what the inputs look like
+
+Frankly, I find it easiest to simply run the following python script to dump the inputs:
+
+```python
+import onnxruntime
+
+session = onnxruntime.InferenceSession("model.onnx", None)
```
Comment (AmosLewis) [inline e2eshark/onnx/models/README.md:39]: Where can I find the file `model.onnx` For a new model?

Comment (zjgarvey) [reply e2eshark/onnx/models/README.md:39]: The llama model kumar mentioned would be in azure (it might be in secure storage).

Comment (zjgarvey) [reply e2eshark/onnx/models/README.md:39]: It might also not be there yet

Comment (AmosLewis) [reply e2eshark/onnx/models/README.md:39]: got it

**Review by AmosLewis — APPROVED** [review]

**Review by kumardeepakamd — APPROVED** [review]
Looks good. Very useful.

**Pull request merged**